packages/report-generation/report_generation/utils.py

```diff
@@ -1,8 +1,12 @@
-import json
 import requests
 from bs4 import BeautifulSoup
 import re
 from .constants import property_to_metrics, property_to_regulations
+from llm_insights.insights import init_llm, metric_insights
+import os
+import dotenv
+
+dotenv.load_dotenv()
```

Comment:
Dotenv load will now run _every time_ this module is loaded -> this is bad, instead the user of the report generation library should pass any variables as keyword arguments, which they themselves can get from a dotenv file